groups/groups.yaml

```diff
@@ -387,6 +387,7 @@ groups:
       - janluk@google.com
       - jkaniuk@google.com
       - jprzychodzen@google.com
+      - jupblb@google.com
```

Comment:
Since you touch this file - can you please remove janluk@ ?

Please also add marcelzieba@google.com

Reply:
Done.